client/my-sites/beta-testing/main.jsx

```diff
@@ -0,0 +1,187 @@
+import {
+	getPostsForQueryIgnoringPage,
+	isRequestingPostsForQueryIgnoringPage,
+} from 'calypso/state/posts/selectors';
+import QueryPosts from 'calypso/components/data/query-posts';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import CardHeading from 'calypso/components/card-heading';
+
+const query = {
+	author: null,
+	number: 3,
+	order: 'DESC',
+	status: 'publish',
+	type: 'post',
+	category: 'testing',
+};
+const horizonSiteId = 90972941;
+
+const BetaTesting = ( { siteId, posts, isRequestingPosts, trackViewHorizonAction } ) => {
+	const translate = useTranslate();
+	const header = (
+		<div className="beta-testing__heading">
+			<FormattedHeader
+				brandFont
+				headerText={ translate( 'Beta Testing' ) }
+				subHeaderText={ translate( 'Join the WordPress.com Beta Community' ) }
+				align="left"
+			/>
+			<div className="beta-testing__view-horizon-button">
+				<Button href="https://horizon.wordpress.com/" onClick={ trackViewHorizonAction }>
+					{ translate( 'Visit Horizon' ) }
+				</Button>
+			</div>
+		</div>
+	);
+
+	const main = (
+		<div className="beta-testing__main">
+			<Card>
+				<CardHeading tagName="h1" size={ 21 }>
+					{ translate( 'Help shape the future of online publishing.' ) }
+				</CardHeading>
+
+				<p>
+					{ translate(
+						"We’re making the web a better place, one release at a time -- and we’re asking for your help. WordPress.com ships updates and improvements every single day, and beta users will get to experience some of these before anyone else. By taking part in this program, you will be helping to shape the future of WordPress.com, making it better for you, your readers, your fellow bloggers, and for millions of people worldwide. You won't be testing everything but there are some important ideas and projects we’d love to share with you. It’s also worth noting that not everything you see will make it into WordPress.com or work exactly the same way once updates go public."
+					) }
+				</p>
+			</Card>
+			<Card>
+				<CardHeading className="beta-testing__heading" tagName="h1" size={ 21 }>
+					<span>{ translate( 'Recent Beta Features' ) }</span>
+					<FollowButtonContainer siteUrl="https://horizonfeedback.wordpress.com" />
+				</CardHeading>
+
+				<div className="beta-testing__posts">
+					{ isRequestingPosts && ! posts && (
+						<article className="beta-testing__post is-placeholder">
+							<CardHeading tagName="h4" size={ 16 }></CardHeading>
+							<p></p>
+							<p></p>
+							<p></p>
+						</article>
+					) }
+					{ posts?.map( ( post ) => {
+						return (
+							<article key={ post.ID } className="beta-testing__post">
+								<CardHeading tagName="h4" size={ 16 }>
+									<a href={ post.URL }>{ post.title }</a>
+								</CardHeading>
+								<p>{ post.excerpt }</p>
+							</article>
+						);
+					} ) }
+				</div>
+			</Card>
+
+			<Card>
+				<CardHeading tagName="h1" size={ 21 }>
+					{ translate( 'How it Works' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **10** times:
`translate( 'How it works.' )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/beta-testing)